### DISCUSSION TITLE
Multiplayer spectate looks wrong with some storyboards

### DISCUSSION THREAD

**Discussion by muffyn** [ppy/osu#15826] [Q&A] [answered]
Many storyboards assume the background will be a black screen as it automatically is everywhere else. Multiplayer spectate does not render a black screen behind the storyboard, which can look bad and cause multiple users' gameplays to overlap.
I think the black screen should always be rendered behind multiplayer spectate windows.

https://streamable.com/8q9fyx

Example map:
https://osu.ppy.sh/beatmapsets/694402#osu/1501410

**Comment by bdach** [comment, marked as answer]
#15827
